osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTail.cs

```diff
@@ -125,36 +121,7 @@ protected override void UpdateHitStateTransforms(ArmedState state)
             }
         }
 
-        protected override void CheckForResult(bool userTriggered, double timeOffset)
-        {
-            if (userTriggered)
```

Comment:
Not sure what this test was doing but better be safe than sorry:
Is this test no longer needed ?

Reply:
`userTriggered = true` comes from user input, if somewhere `UpdateResult(true)` was called.

It's irrelevant here because that's not called.